src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts

```diff
@@ -77,9 +99,20 @@ export class UmbDocumentTypeWorkspaceViewStructureElement extends UmbLitElement
 				</umb-property-layout>
 			</uui-box>
 			<uui-box headline="Presentation">
```

Comment:
Not part of this PR, but could you add some localization here and maybe in the file in general?

Reply:
Should I do as part of this PR, or separate PR? Just thinking whether we want the feature in preview008 or for beta001?

Reply:
New PR is fine 👍